setup.py

```diff
@@ -0,0 +1,24 @@
+from setuptools import find_packages, setup
+
+setup(
+    author="CS50",
+    author_email="sysadmins@cs50.harvard.edu",
+    classifiers=[
+        "Intended Audience :: Education",
+        "Programming Language :: Python :: 3",
+        "Topic :: Education",
+        "Topic :: Utilities"
+    ],
+    description="This is check50, with which you can check solutions to \
+        problems for CS50.",
+    install_requires=["argparse", "pexpect", "requests", "backports.shutil_which", "termcolor"],
+    keywords=["check", "check50"],
+    name="check50",
+    py_modules=["check50", "config"],
```

Comment:
Yeah, per https://docs.python.org/3/faq/programming.html#how-do-i-share-global-variables-across-modules, it's still the canonical way to share variables between different modules.